salt_stack/salt/top.sls

```diff
@@ -57,11 +61,11 @@ base:
         - scalyr
         - scalyr.update_host
         - git
+        - chrony
 
     'activities*':
         - scalyr
         - scalyr.update_host
-        - open-files.increase-open-files
```

Comment:
This looks like something that was incorrectly removed by the merge? @sandyhider